gallery-so/gallery
# Mobile Mint Link

### Summary of Changes

- Add mint link input to post composer
- Show mint link to feed, nft detail screen and collection screen

### Note

- [x] Sync with backend on how to pass overwrite URL

### Demo or Before/After Pics

**Post composer**

https://github.com/gallery-so/gallery/assets/4480258/4abc1b5e-a9b2-4ad1-87c3-58e6a191e070


**Mint Button CTA**

https://github.com/gallery-so/gallery/assets/4480258/b4b2ba36-c484-4ff2-a6e1-83822caabe01



### Edge Cases

1. Overwrite mint url in the post composer that not mint fun & zora
2. The token from omnibus contract shouldn't show mint button
3. Only show mint button on eth, arbi, zora and base
4. Add referrer address to the mint link

### Testing Steps

1. Create any post and you should see a mint link toggle button
2. You should see the mint button in the feed / nft detail screen / collection if there is support link


### Checklist

Please make sure to review and check all of the following:
